portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACApi.macro

```diff
@@ -1,5 +1,18 @@
 definition {
 
+	macro assertApiResponse {
+		for (var expected : list ${expectedList}) {
+			if (contains(${apiDataResponse}, ${expected})) {
```

Comment:
Hey @manuferreira, `apiResponse` is a `static var` so you can use this variable here in your condition and you don't need to pass a new variable with the value of `apiResponse` in the parameter of this macro

Reply:
Actually, I do. Because I was getting issues in poshi, if I set the apiResponse by default in this macro instead of the apiDataResponse (that is not an static variable), and if I want to pass another response in this function that is not the default apiResponse , I will get an:
```
    [exec] com.liferay.poshi.runner.PoshiRunner > test[LocalFile.DataReportAPI#RequestToAccessDataForSpecificBlog] STANDARD_ERROR
     [exec]     java.lang.Exception: Unable to set var 'apiResponse' as parameter of function. It is already set in the static context.
```
That's why I changed this macro not to use the apiResponse static variable UNLESS we want to. This way we can pass any response we want in this macro and we won't be limited to remain in the static context.

Reply:
Ohhh nice manu, it makes sense to do this for the macro to be more reusable. Thanks 🎉